innvestigate/tests/tools/test_perturbate.py

```diff
@@ -26,15 +27,75 @@
 def test_precommit__PerturbationAnalysis():
     # Some test data
     if keras.backend.image_data_format() == "channels_first":
-        input_shape = (10, 1, 28, 18)
+        input_shape = (2, 1, 4, 4)
     else:
-        input_shape = (10, 28, 18, 1)
-    x = np.random.rand(*input_shape)
+        input_shape = (2, 4, 4, 1)
+    x = np.arange(2 * 4 * 4).reshape(input_shape)
```

Comment:
Maybe we can do replace `2*4*4` with `np.product(input_shape)` ? So we don't have to change the value again if the dimensions of `input_shape`?

Reply:
Sounds good!